fidesctl/src/fidesapi/migrations/versions/fdd28c39a679_add_responsibility_role_attribute_for_.py

```diff
@@ -0,0 +1,30 @@
+"""add responsibility_role attribute for system
+
+Revision ID: fdd28c39a679
+Revises: 7a94527643a1
+Create Date: 2022-03-21 17:37:23.855761
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "fdd28c39a679"
+down_revision = "7a94527643a1"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.add_column(
+        "systems", sa.Column("responsibility_role", sa.String(), nullable=True)
```

Comment:
i dont think this should be nullable?

Reply:
Agreed, forgot that we set a default for `controller` 👍🏽  - nice catch!